etc/config-template.toml

```diff
@@ -38,6 +38,9 @@ labels = {}
 [storage]
 # set the path to rocksdb directory.
 data-dir = "/tmp/tikv/store"
+# set the path to raft rocksdb directory, default value is data-dir/raft
+# raft-db-path = "/tmp/tikv/store/raft"
```

Comment:
we should let the users assign the separated path if they want save raft and kv in different disks.